.goreleaser.yaml

```diff
@@ -55,21 +55,14 @@ brews:
       zsh_completion.install "completions/dlp-cli.zsh" => "_dlp-cli"
       fish_completion.install "completions/dlp-cli.fish"
 
-winget:
+scoops:
```

Comment:
this isnt a typo? @dwu359

Reply:
No, scoops fits our use case much better than winget rn, so I'm probably just going to delete anything winget related in this pr or in the future